README.md

```diff
@@ -1,20 +1,26 @@
 # cell-types-analysis
-A suite of scripts for analysis of cell type classification tools outputs. 
+A suite of scripts for analysis of scRNA-seq cell type classification tool outputs. These can be used both for evaluating the existing methods by running pipelines on labelled data and for analysing predicted labels for novel data sets.  
 
-## Usage 
+## Usage
+
+### Tool evaluation scenario 
 For each tool, a list of statistics is generated. The following metrics are used:
 * percentage of unlabelled cells
 * change in the percentage of unlabelled cells in predicted labels vs reference labels
 * proportion of exact matches between predicted and reference labels (exact matching)
 * average proportion of shared words between reference and predicted labels (partial matching)
 * accuracy & median F1-score
 * average semantic similarity between predicted and reference labels
+* combined score: combination of the above metrics expressed as a single number. This score is then used as a 'weight' for labels predicted by corresponding tool in production scenario. 
+
+For a subset of statistics, empirical distribution function can be estimated and then used to determine the p-values of computed scores.  
 
-For a subset of statistics, empirical distribution function can be estimated and then used to determine the p-value of obtained scores.  
+A common output format across tools is assumed - a tab-separated table with 3 columns: cell id (or barcode), predicted labels, and corresponding prediction scores (e.g. p-values or distance metrics). In cases when scores cannot be retrieved, this column can be omitted. Outputs from multiple tools must be stored in a single directory, with file names prefixed by tool name, e.g. `toolX_output.tsv`. A reference table is required, with the following compulsory columns: cell ids, reference labels, and corresponding cell ontology terms.
```

Comment:
Can you add a few lines of example output here 

```
Like this
and this
```

Reply:
@pinin4fjords  will it be ok if I add a directory with example inputs (we already have example output) and point out there?

Reply:
That's good too, but for reading documentation it's nice to have a snippet inline by way of illustration too, so please do that.

Reply:
Doesn't need to be big, just a few lines to illustrate the format.

Reply:
okay, can do that